src/app/features/home/post-capture-tab/post-capture-tab.component.ts

```diff
@@ -30,6 +30,8 @@ export class PostCaptureTabComponent implements OnInit {
     this._focus$.next(focus);
   }
 
+  categories!: string;
```

Comment:
Avoiding asserting variable declaration. Assign `'Photo'` to it as default value.